bootstrap_bootkube_log.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$(dirname $0)/show_bootstrap_log.sh bootkube.service
```

Comment:
Was this supposed to be included here?

Reply:
Yeah, I needed that to debug the other change so I just left it in this PR. It's a separate commit for tracking later.

Reply:
It appears not unidentical to this other one though: https://github.com/openshift-metal3/dev-scripts/pull/763/files#diff-19647512eab29fd4eda6d2f9dc694ee7